node/windows-packaging/CalicoWindows/README.txt

```diff
@@ -1,9 +1,9 @@
-Calico for Windows

```

Comment:
Could you revert the change to this file? It looks like it wasn't actually modified.

Reply:
I try it